shared/src/types.rs

```diff
@@ -348,6 +348,18 @@ pub struct RenamePeerOpts {
     pub yes: bool,
 }
 
+
+#[derive(Debug, Clone, PartialEq, Eq, Args)]
+pub struct EnableDisablePeerOpts {
+    /// Name of peer to enable/disable
+    #[clap(long)]
+    pub name: Option<Hostname>,
+
+    /// Bypass confirmation
+    #[clap(long)]
+    pub yes: bool,
```

Comment:
Am I correct in that the `yes` flag can only really be effectively used alongside the `name` flag? In that case we may want to add a [`requires`](https://docs.rs/clap/4.1.6/clap/builder/struct.Arg.html#method.requires) dependency, though I'm not sure if we can express that with the clap-derive syntax.

Reply:
Per the [docs](https://docs.rs/clap/latest/clap/_derive/#terminology) it should be doable with a mere

```rust 
    #[arg(long, requires("name"))]
    pub yes: bool,
```

Reply:
Thanks, I made the change accordingly.